session/pessimistic_test.go

```diff
@@ -1162,3 +1162,37 @@ func (s *testPessimisticSuite) TestRCSubQuery(c *C) {
 	tk.MustQuery("select * from t1 where c1 = (select c1 from t where c1 = 1) and 1=1;").Check(testkit.Rows("1 4"))
 	tk.MustExec("rollback")
 }
+
+func (s *testPessimisticSuite) TestGenerateColPointGet(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
```

Comment:
how about help a test case

```
set global tidb_row_format_version=1;
````
then re-run those script to ensure it also work for old row format